### DISCUSSION TITLE
Enterprise Roundup October 2024

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#142394] [Enterprise]
![Header of Enterprise category features the text Enterprise Roundup on the upper lefthand corner on a black background with an abstract peach and teal abstract line design](https://github.com/user-attachments/assets/7f286656-ba58-49ef-857a-4bd9ea7e4c4c)

We are less than a week away from GitHub Universe 2024 and want to make sure you are as ready to celebrate as we are. Plus, more exciting updates from around the Community and the platform. Let’s ship this 🚢

### 🪐 Universe 2024: As the world’s fair of software, GitHub Universe 2024 will be an unparalleled gathering of the brightest minds, companies, and innovators in the industry. 
* **Get Started with Universe 2024**: With less than a week to go, join us in everything Universe on the community. Check out [our discussion](https://github.com/orgs/community/discussions/141919) to start celebrating. 
* **San Francisco Survival Guide (and recommendation post!)**: Heading to San Francisco to join us in-peron? SF local @novium2 has you covered for food, sights, and packing for the city by the bay in [her discussion](https://github.com/orgs/community/discussions/141921).
* **Register for virtual and in-person**: It’s not too late to reserve your spot! [View the agenda](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog?tab.day=20241029) and [buy your tickets now](https://githubuniverse.com/#tickets).

### 🔍 Curious Questions:  Quality questions asked in the past month! 
* **Interested in understanding how billing works via AzureCSP** - [check out the answer in this discussion](https://github.com/orgs/community/discussions/141027). Thank you @mana7331 for asking. 
* [What is the cause of "The XXX organization cannot be invited because it has marketplace applications"?](https://github.com/orgs/community/discussions/139255)  Asked and solved by @OndrejSpanel! 

### 🗳️ Recent Releases: All updates, info, releases, etc that are Enterprise Server and Cloud -related!
* **Discover the GitHub Enterprise Cloud FAQ in the New GitHub Trust Center**: Now you can find answers to commonly asked questions about GitHub Enterprise Cloud in the GitHub Trust Center, a comprehensive resource for understanding how GitHub meets security, privacy, and compliance standards. [Review the highlights in our changelog post](https://github.blog/changelog/2024-10-01-discover-the-github-enterprise-cloud-faq-in-the-new-github-trust-center/). 
* **Copilot content exclusions now available for enterprise admins**: Enterprise admins can now manage and apply content exclusions at the enterprise level. This expands upon previous capabilities where only org admins and repo admins could apply exclusions. [Learn how to get started via the changelog](https://github.blog/changelog/2024-10-08-copilot-content-exclusions-now-available-for-enterprise-admins/). 
* **Bring Your Own Identity Provider to Enterprise Managed Users**: GitHub Enterprise Cloud’s open support for the System for Cross-domain Identity Management (SCIM) specification is now generally available for Enterprise Managed Users (EMUs). [Get all the details in the changelog](https://github.blog/changelog/2024-09-26-bring-your-own-identity-provider-to-enterprise-managed-users/). 

### 📣 Announcements: The latest updates around the Community and GitHub 🆕
* **New Terminology for GitHub Previews**: @sharipaltrowitz shares that we’re simplifying the terminology we use to refer to products that are in testing and validation stages. That means, you’ll start seeing the word “Preview” instead of “Alpha” or “Beta” to describe our features that are not yet generally available. [Read her discussion post to learn more](https://github.com/orgs/community/discussions/141918). 

I’ll be at GitHub Universe along with @ossanna16 and @ettaboyle. Come stop by the GitHub Experts Center in the Gateway Pavilon to say hello, ask questions, or get GitHub best practices/advice. What are you looking forward to at Universe?

**Comment by Terastvslet** [comment]
WLFI

**Comment by queenofcorgis** [comment]
Get all the latest questions and releases in [November's Roundup discussion](https://github.com/orgs/community/discussions/145144) - now live!
